_docs/quickstarts/scanner/descriptions.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Scanner Descriptions and Dependencies
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/howto/configure/configuring-forseti#configuring-scanner %}).
+
+Each scanner depends on one or more Forseti Inventory pipelines that must be set
+to `enabled: true` in your `forseti_conf.yaml` file. Learn more about
+[configuring
+Inventory]({% link _docs/howto/configure/configuring-forseti#configuring-inventory %}).
+
+## bigquery scanner
+
+BigQuery datasets have access properties that can publicly expose your datasets.
+The BigQuery scanner supports a blacklist mode, to ensure unauthorized users
+don't gain access to your datasets.
+
+For examples of how to define scanner rules for your BigQuery datasets, see the
+[bigquery_rules.yaml](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/bigquery_rules.yaml)
+rule file.
+
+The bigquery scanner depends on the following Forseti Inventory pipelines:
+
+-   `load_bigquery_datasets_pipeline`
```

Comment:
The standard so far is once space between the ordered list and the list item itself, e.g.

`- `load_bigquery_datasets_pipeline``